cluster/ephemeral_provider.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+function _kubectl() {
+  export KUBECONFIG=cluster/$KUBEVIRT_PROVIDER/.kubeconfig
+  cluster/.kubectl "$@"
+}
+
+function seed_images(){
+  container=""
+  container_alias=""
+  images="${@:-${DOCKER_IMAGES}}"
+  for arg in $images; do
+      name=$(basename $arg)
+      container="${container} registry:5000/${name}:latest"
+  done
+
+  # We don't need to seed the nodes, but in the case of the default dev setup we'll just leave this here
+  for i in $(seq 1 ${KUBEVIRT_NUM_NODES}); do
```

Comment:
Just and FYI why this code was in here in the first place. This is to make sure the nodes have the latest version of the containers in case the pull policy is not set to always pull.

Reply:
ack, I'll likely point that out in the comment for future John :)